src/main/java/picard/sam/markduplicates/MarkDuplicates.java

```diff
@@ -234,7 +234,9 @@ protected int doWork() {
         final SAMFileHeader.SortOrder sortOrder = header.getSortOrder();
 
         final SAMFileHeader outputHeader = header.clone();
-
+        if(ASSUME_SORTED || ASSUME_SORT_ORDER != null) {
+            outputHeader.setSortOrder(SAMFileHeader.SortOrder.unsorted);
+        }
```

Comment:
why is the output always unsorted?  I thought that `MarkDuplicates` can still take in and output coordinate sorted BAMs?  Also `if(` -> `if (`.